CONTRIBUTING.md

```diff
@@ -1,5 +1,50 @@
 # Contributing to Presto
 
+Thanks for your interest in Presto.  Our goal is to build a fast, scalable and reliable distributed SQL query engine for running interactive analytic queries against data sources of all sizes ranging from gigabytes to petabytes.
```

Comment:
Our focus is no longer limited to interactive. We are clearly heavily into batch analytics for some time now :-P
```suggestion
Thanks for your interest in Presto.  Our goal is to build a fast, scalable and reliable distributed SQL query engine for running low latency interactive as well as batch analytic queries against data sources of all sizes ranging from gigabytes to petabytes.
```

Reply:
Tweaked it to be similar to that.